SimTracker/SiPhase2Digitizer/plugins/Phase2TrackerDigitizerAlgorithm.cc

```diff
@@ -891,8 +891,12 @@ void Phase2TrackerDigitizerAlgorithm::digitize(const Phase2TrackerGeomDetUnit* p
     add_noise(pixdet);  // generate noise
   if (addXtalk_)
     add_cross_talk(pixdet);
-  if (addNoisyPixels_)
-    add_noisy_cells(pixdet, theHIPThresholdInE / theElectronPerADC_);
+  if (addNoisyPixels_) {
+    float thresholdInNoiseUnits = 99.9;
```

Comment:
@suchandradutta , should this value be in the list of parameters?, is it const or may be era dependent?

Reply:
@civanch this is not era dependent. This a very large sigma value (somewhat arbitrary) to protect the add_noisy_cells function when the channel noise is zero.